File: Assignment 1/code/code.c
//code to verify the trigonometric idetity for a few values of theta

#include <stdio.h>
#include <math.h>

int main()
{
float x;//x to store sin(theta)
float y;//y to store cos(theta)
float z;//z to store tan(theta)
printf("Value of trigonometric expression at \ndifferent values of theta\n");
for(float i =0.3;i<2*M_PI;i=i+.3)
{	
	x = sin(i);
	y = cos(i);
	z = tan(i);
	float expression = (1+ (1/z) -(1/x))*(1+z+(1/y));
	printf("theta = %f , expression = %f\n",i,expression);
}
}

File: Assignment 1/code/code.py
import numpy as np

for i in range(1,20):
    	
	#input parameter
    
	x=i*0.3
	y = (1+1/np.tan(x)-1/np.sin(x))*(1+np.tan(x)+1/np.cos(x))
        
       #output
       
	print(y)
